src/FakeItEasy/Configuration/AnyCallCallRule.cs

```diff
@@ -8,29 +8,21 @@ internal class AnyCallCallRule
     {
         private Func<ArgumentCollection, bool> argumentsPredicate = x => true;
         private Type applicableToMembersWithReturnType = typeof(AllReturnTypes);
+        private readonly FakeManager manager;

```

Comment:
Since I'm already asking for a change in `CallConstraintDescriber` below, would you mind moving `manager` up to line 9? Having a readonly field after a non-readonly field [causes an warning in the build](https://ci.appveyor.com/project/FakeItEasy/fakeiteasy/builds/42130140/job/0xgev6ijlyhomatl#L68).